virttest/qemu_monitor.py

```diff
@@ -492,7 +492,7 @@ def is_responsive(self):
         try:
             self.verify_responsive()
             return True
-        except MonitorError:
+        except (MonitorError, ValueError):
```

Comment:
Can we simply make it by checking the `fileno`, in this case?
```python
if self._socket.fileno() < 0:
    return False
```

Reply:
Good idea, will test and come back to you here.

Reply:
This suggestion works and is more refined, thanks! I pushed the changes and have tested them locally already.